docs/reference/truststore-types.md

```diff
@@ -0,0 +1,45 @@
+# Support different types of trust store
+
+Notation spec supports 3 trust store types: ca, tsa and signingAuthority: https://github.com/notaryproject/specifications/blob/main/specs/trust-store-trust-policy.md#trust-policy-properties
+
+Ratify previously didn't differentiate between types, and we used to just set it as ca type as default. As we already have users use signingAuthority instead of ca type, we provide support to this feature. And we would also support tsa for tsa signature.
+
+# Implementation
```

Comment:
Is this doc a design doc or user manual? If this is a design doc, it should be submitted to https://github.com/deislabs/ratify/tree/staging/docs